src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
+$gap-size: 1px;
+$ceil-width: 100px;
+$ceit-height: 100px;
+$days: (
+  "mon": 0,
+  "tue": 1,
+  "wed": 2,
+  "thu": 3,
+  "fri": 4,
+  "sat": 5,
+  "sun": 6,
+);
+
+$month-length: 28, 29, 30, 31;
+
+html {
+  font-size: 30px;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  margin: 0 auto;
```

Comment:
BEM-block shouldn't have any margins, it you want to specify them use BEM mix instead.